examples/components/utils/aggregatemodelweights/spec.yaml

```diff
@@ -44,7 +52,8 @@ command: >-
   $[[${{inputs.input_silo_3}}]] 
   $[[${{inputs.input_silo_4}}]] 
   $[[${{inputs.input_silo_5}}]] 
-
+  --ancillary_files $[[${{inputs.ancillary_files}}]] 
```

Comment:
you will have to surround `--ancillary_files` as well in the `$[[ ... ]]` block

Reply:
that's right, good catch!